diff -rupN python/setup.py python/setup.py
--- python/setup.py
+++ python/setup.py
@@ -7,6 +7,14 @@ from platform import system
 from numpy import get_include
 from numpy.distutils.system_info import get_info, BlasNotFoundError 
 
+def get_info(x):
+    import os
+    return {
+        'include_dirs': [os.environ.get('LAPACK_INCLUDE_PATH', '')],
+        'library_dirs': [os.environ.get('LAPACK_LIB_PATH', '')],
+        'libraries': os.environ.get('LAPACK_LIBS', '').split(),
+    }
+
 def install_scs(USE_64_BIT_BLAS, blas_info, lapack_info, USE_OPENMP, rootDir): 
     libraries = []
     if system() == 'Linux':
